bin/lint-markdown-api-history.ts

```diff
@@ -267,6 +308,27 @@ async function main(
           continue;
         }
 
+        if (disallowComments) {
+          const numberOfHashtagsInCodeBlock = (codeBlock.value.match(/#/g) || []).length;
+          const unsafeHistoryJsonString = JSON.stringify(unsafeHistory);
+          const numberOfHashtagsInHistoryJsonString = (unsafeHistoryJsonString.match(/#/g) || [])
+            .length;
+
+          // If the number of hashtags in the code block is greater than the number of hashtags in the JSON string,
+          //  then the API History contains comments.
+          if (numberOfHashtagsInCodeBlock > numberOfHashtagsInHistoryJsonString) {
```

Comment:
This is simple and works, but maybe something like this is more robust:

```js
YAML.visit(parseDocument(codeBlock.value), (...args) => args.forEach(e =>
	(e?.comment || e?.commentBefore) &&
	throw new Error("found comment");
))
```

I'll look into the performance.

Reply:
Switched to traversing the tree in [`fbdef15` (#81)](https://github.com/electron/lint-roller/pull/81/commits/fbdef1543b2989eea92d4e28095e946370b41fb1) instead of counting `#`. Didn't measure a noticeable performance difference.

Reply:
Seems reasonable to me. 👍